clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -219,33 +219,39 @@ def drain_node(self, node_name: str, disable_eviction: bool) -> bool:
             return False
 
     def cordon_node(self, node_name: str) -> bool:
+        now = str(arrow.now().timestamp)
         try:
-            self._core_api.patch_node(
-                name=node_name,
-                body={
-                    "spec": {
-                        "unschedulable": True,
-                    },
-                },
+            ip = self._get_ip_from_node_name(node_name)
+            node = self._nodes_by_ip[ip] if ip in self._nodes_by_ip else None
```

Comment:
This is the same as
```suggestion
            node = self._nodes_by_ip.get(ip)
```

(same below)